perception/image_projection_based_fusion/include/image_projection_based_fusion/fusion_node.hpp

```diff
@@ -76,30 +83,40 @@ class FusionNode : public rclcpp::Node
 
   void publish(const Msg & output_msg);
 
+  void timer_callback();
+  void setPeriod(const int64_t new_period);
+
   std::size_t rois_number_{1};
   tf2_ros::Buffer tf_buffer_;
   tf2_ros::TransformListener tf_listener_;
 
   // camera_info
   std::map<std::size_t, sensor_msgs::msg::CameraInfo> camera_info_map_;
   std::vector<rclcpp::Subscription<sensor_msgs::msg::CameraInfo>::SharedPtr> camera_info_subs_;
+  std::vector<double> tan_h_;  // horizontal field of view
+
+  /** \brief The maximum number of messages that we can store in the queue. */
+  int maximum_queue_size_ = 3;
 
-  // fusion
-  typename message_filters::Subscriber<Msg> sub_;
-  message_filters::PassThrough<DetectedObjectsWithFeature> passthrough_;
-  std::vector<std::shared_ptr<message_filters::Subscriber<DetectedObjectsWithFeature>>> rois_subs_;
-  inline void dummyCallback(DetectedObjectsWithFeature::ConstSharedPtr input)
-  {
-    passthrough_.add(input);
-  }
-  using SyncPolicy = message_filters::sync_policies::ApproximateTime<
-    Msg, DetectedObjectsWithFeature, DetectedObjectsWithFeature, DetectedObjectsWithFeature,
-    DetectedObjectsWithFeature, DetectedObjectsWithFeature, DetectedObjectsWithFeature,
-    DetectedObjectsWithFeature, DetectedObjectsWithFeature>;
-  using Sync = message_filters::Synchronizer<SyncPolicy>;
-  typename std::shared_ptr<Sync> sync_ptr_;
-
-  // output
+  rclcpp::TimerBase::SharedPtr timer_;
+  double timeout_ms_ = 10;
+  int match_threshold_ms_ = 50;
```

Comment:
```suggestion
  double timeout_ms_{};
  double match_threshold_ms_{};
```
[imo] do not declare default parameters to avoid confusion, since they are different in this PR. And use double for `match_threshold_ms_` to match other `*ms_` params.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2175/commits/7807a448f3c772d49ebe50bc6f5fe3863ead6a64